Whiskey/en-US/about_Whiskey_GitHubRelease_Task.help.txt

```diff
@@ -0,0 +1,71 @@
+    
+    The release is created on the head (i.e. most recent commit) on the master branch. Use the `Commitish` property to change the release's commit. We recommend using the `$(WHISKEY_SCM_COMMIT_ID)` variable, which is the commit ID currently building, but any valid Git [revision](https://git-scm.com/docs/gitrevisions) should be acceptable. 
+
+    GitHub uses the commit message of the release's commit as the release name. Use the `Name` property to set your own name.
+
+    Use the `Description` property to describe the release. This is a good place for release notes.
+
+    You can upload assets/files, too. These files are visible on the release's page in GitHub and are downloadable by Users. Use the `Assets` property to upload files. `Assets` should be a list of mappings. Each mapping must have a `Path` property (which is a path, relative to the whiskey.yml file, to a file) and a `ContentType` property, which is the [media type of the file(https://www.iana.org/assignments/media-types/media-types.xhtml) (e.g. `application/json`). By default, the file's name is used as its name in GitHub. To customize the file's name, use the asset's `Name` property.
+
+    If your repository in GitHub doesn't already have the release's tag, a new release is created. If the tag already exists, the release's name and description are updated. If an file already exists, its name is updated.
+
+    This task enables TLS 1.2 in the PowerShell process running the build.
+
+PROPERTIES
+    * `RepositoryName` (***mandatory***): the owner and name of the repository, as a URI path, e.g. `OWNER/NAME`.
+    * `Tag` (***mandatory***): the tag name to create for the release. An actual Git tag will be created in your repository.
+    * `ApiKeyID` (***mandatory***): The ID of the API key to use when making GitHub API calls. Add API keys to your build with the `Add-WhiskeyApiKey` function. You can generate an access token on your [GitHub tokens settings page](https://github.com/settings/tokens).
+    * `Commitish`: the commitish that identifies the commit you're releasing. By default, the GitHub API uses the head (i.e. most recent commit) of the master branch. We recommend using the `$(WHISKEY_SCM_COMMIT_ID)` variable, which is the commit ID currently building.
+    * `Name`: the name of the release. By default, the GitHub API uses the commit message of the commit you're releasing.
+    * `Description`: a description of the release. This is a good place to put release notes.
+    * `Assets`: a list of files to upload. These files will show on the release's page in GitHub and users will be able to download them. This is a good place to upload a ZIP file or your application. Each asset is a mapping with these properties:

```

Comment:
Typo (I think?): "upload a ZIP file **or** your application" should be "upload a ZIP file **of** your application"